elements/progress-slider/progress-slider.js

```diff
@@ -82,10 +84,10 @@ let self = class ProgressSlider extends HTMLElement {
 		let value = this[source + "Element"].value;
 
 		if (source === "slider") {
-			this.valueElement.value = value;
+			this.valueElement.value = this.show === "progress" ? this.progressAt(value) : value;
 		}
 		else if (source === "value") {
-			this.sliderElement.value = value;
+			this.sliderElement.value = this.show === "progress" ? this.valueAt(value) : value;
```

Comment:
Nope, the slider value is always what it is, it's only the spinner that may transform it.

Reply:
I might be missing something, then. If we are here, it means that the spinner is the source of the value change, and we need to update the slider value accordingly. Since the spinner value might show progress, we should take it into account.

Reply:
Oops, you’re right, I misread this and thought you were doing something different!